forest/Cargo.toml

```
@@ -70,6 +70,7 @@ rocksdb = ["db/rocksdb", "ipld_blockstore/rocksdb"]
 sled = ["db/sled", "ipld_blockstore/sled"]
 devnet = ["actor/devnet", "networks/devnet", "testnet"]
 interopnet = ["actor/interopnet", "networks/interopnet", "testnet"]
+calibnet = ["networks/calibnet"]
```

When you have defined the features in the `actor` and `networks` crate, you need to enable them here. 
```suggestion
calibnet = ["networks/calibnet", "networks/calibnet", "testnet"]
```

---

:+1: